apps/concept-catalog/app/actions/concept/actions.ts

```diff
@@ -4,7 +4,7 @@ import {
   deleteConcept as deleteConceptApi,
   createConcept as createConceptApi,
   patchConcept as patchConceptApi,
-  getConcept,
+  getConcept, removeImportResultConcept as removeImportResult,
```

Comment:
Bør vel bare kjøre `yarn format`, så fikser den alt av disse tingene
```suggestion
  getConcept,
  removeImportResultConcept as removeImportResult,
```